### PR TITLE
KUZBO-132 watch data

### PR DESCRIPTION
**Pull request by jenow**

### CODE REVIEW

File: src/components/Materialize/collapsible.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export default {
+  id: 'collapsible',
+  bind: () => {
+    setTimeout(() => {
+      /* eslint no-undef: 0 */
```
Comment (scottinet) [inline src/components/Materialize/collapsible.js]: This eslint pragma disables the linter for all subsequent lines of the file. From the position of this comment, I believe what you meant was: `// eslint-disable-next-line no-undef`

File: src/vuex/modules/data/actions.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import kuzzle from '../../../services/kuzzle'
-import {RECEIVE_MAPPING, RECEIVE_INDEXES_COLLECTIONS} from './mutation-types'
+import {RECEIVE_MAPPING, RECEIVE_INDEXES_COLLECTIONS, ADD_NOTIFICATION, EMPTY_NOTIFICATION} from './mutation-types'
 import {SET_ERROR} from '../common/mutation-types'
 import Promise from 'bluebird'
 
+let room
```
Comment (scottinet) [inline src/vuex/modules/data/actions.js]: As seen together, this global variable should instead be put in the store.

File: test/unit/specs/data/actions.spec.js [hunk 3]
```diff
@@ -8,14 +8,14 @@ describe('listIndexesAndCollections action', () => {
     '../../../services/kuzzle': {
       listIndexes (cb) {
         if (triggerError[0]) {
-          cb({message: 'error'})
+          cb({message: 'error', stack: 'stack'})
```
Comment (scottinet) [inline test/unit/specs/data/actions.spec.js]: This shouldn't be necessary since bluebird 3.3

File: src/vuex/modules/data/store.js [hunk 4]
```diff
@@ -14,6 +19,15 @@ export const mutations = {
   },
   [RECEIVE_MAPPING] (state, mapping) {
     state.mapping = mapping
+  },
+  [ADD_NOTIFICATION] (state, notification) {
+    state.notifications.push(notification)
```
Comment (xbill82) [inline src/vuex/modules/data/store.js]: Shouldn't we limit the size of this array somehow?

**Comment by xbill82** [comment]
A part from the conflicts and my comments, :+1:

File: src/vuex/modules/data/store.js (hunk 4 above)
Comment (jenow) [inline src/vuex/modules/data/store.js]: I was thinking it could be subject to another PR

**Comment by stafyniaksacha** [comment]
+1
